CHANGELOG.md

```diff
@@ -72,6 +72,8 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Add buffered lookahead for Jackson ([#338](https://github.com/opensearch-project/opensearch-java/pull/338))
 - Add support for headers and sort parameters in cat requests ([#413](https://github.com/opensearch-project/opensearch-java/pull/413))
 - Add support for data stream operations ([#419](https://github.com/opensearch-project/opensearch-java/pull/419))
+- Add missing minScore, postFilter, searchAfter, sort, trackScores in the MultisearchBody ([#516](https://github.com/opensearch-project/opensearch-java/pull/516))
+- Add translog option object and missing translog sync interval option in index settings ([#518](https://github.com/opensearch-project/opensearch-java/pull/518))
```

Comment:
Ah ... apologies @Mstrutov , this line should go under `Unreleased 2.x` section

Reply:
Oh, I see, pardon me! I have also added a deprecation line --- please, see if that was done correctly.

Reply:
All good, thanks @Mstrutov !